grails-app/views/stripe/_script.gsp

```diff
@@ -57,6 +57,15 @@ this.stripe_utils = (function() {
 			}
 			jQuery.extend(options, addressInformation)
 		}
+		if(options.number === ''
+			|| options.cvc === ''
+			|| options.exp_month === ''
+			|| options.exp_year === '
+			|| (enableAvs === 'true' && options.address_line1 === '')
+			|| (enableAvs === 'true' && options.address_zip === '')) {
+			failureHandler({ error: { code: 'all.fields.required', message: 'Please fill in all form fields below' }});
```

Comment:
For code reviewer: the default failureHandler cares about `response.error.message`, while our custom one cares about `response.error.code`, hence this setup.